src/sentry/conf/server.py

```diff
@@ -1343,3 +1344,11 @@ def get_raven_config():
 # Relay
 # List of PKs whitelisted by Sentry
 SENTRY_RELAY_WHITELIST_PK = []
+
+# CDN
+# If this is an absolute url like e.g.: https://js.sentry-cdn.com/
+# the full url will look like this: https://js.sentry-cdn.com/<public_key>.min.js
+# otherwise django reverse url lookup will be used.
+JS_SDK_LOADER_CDN_URL = ''
```

Comment:
Don't you want this `JS_SDK_LOADER_CDN_URL = '/js-sdk-loader/'` or something? Otherwise, it'll generate wrong urls.

Reply:
The idea here is to only add a full url that should be used instead of the `reverse` django url see: https://github.com/getsentry/sentry/pull/8640/files/ce88a840cffd2631bd17500b360f101902603752#diff-9d1fb52b257784a8bb2c81b5a0631c76R204

So adding anything instead of a full `http(s)://` url will do nothing, also see your other comment.